features-json/payment-request.json

```diff
@@ -349,7 +349,8 @@
     "5":"Unlike Desktop Chrome, support has been in Chrome for Android since version 53.",
     "6":"Can be enabled via the `dom.payments.request.enabled` flag in \"about:config\" flag since 55.",
     "7":"Missing support for PaymentResponse.prototype.retry() method",
-    "8":"Enabled by default in Nightly, but \"browser.search.region\" flag in \"about:config\" must be set to 'US'"
+    "8":"Enabled by default in Nightly, but \"browser.search.region\" pref in \"about:config\" must be set to 'US'",
```

Comment:
I would suggest removing (8) because it's no longer relevant due to being Nightly-only: you can't change that pref in 64 Beta/Release to get the feature. Users should also never be changing that pref so I'd rather not call it out.

Reply:
Good point.

Reply:
So, let's just replace 8 with 9, yeah?